### PR TITLE
doc: Link fix to DFU document

### PR DESCRIPTION
**Pull request by peknis**
Fixed the link in the Device Firmware Upgrade document.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: doc/guides/device_mgmt/dfu.rst [hunk 1]
```diff
@@ -48,5 +48,5 @@ More detailed information regarding the use of MCUboot with Zephyr  can be found
 in the `MCUboot with Zephyr`_ documentation page on the MCUboot website.
 
 .. _MCUboot boot loader: https://mcuboot.com/
-.. _MCUboot with Zephyr: https://mcuboot.com/documentation/readme-zephyr/
```
Comment (gmarull) [inline doc/guides/device_mgmt/dfu.rst:51]: I can load https://mcuboot.com/documentation/readme-zephyr/ (without www).

Comment (peknis) [reply doc/guides/device_mgmt/dfu.rst:51]: Does it matter if it has www in front?

Comment (gmarull) [reply doc/guides/device_mgmt/dfu.rst:51]: As soon as their webserver handles that, no. It seems to work without in this case. Most websites work with or without www nowadays.

**Comment by peknis** [comment]
Oh, I did not notice this had been fixed already. Closing this PR.

**Pull request closed without merging**